### PR TITLE
prevent defect fix

### PR DESCRIPTION
**Pull request by chunseoklee**
1. iotjs_util.cpp : ftell may return zero
2. iotjs_module_gpio : memory leak of req_wrap

IoT.js-DCO-1.0-Signed-off-by: Chunseok Lee chunseok.lee@samsung.com

### CODE REVIEW

File: src/iotjs_module_gpio.cpp [hunk 1]
```diff
@@ -98,10 +98,12 @@ JHANDLER_FUNCTION(SetPin) {
 
   if (req_data->dir < kGpioDirectionNone ||
       req_data->dir > kGpioDirectionOut) {
+    delete req_wrap;
     JHANDLER_THROW_RETURN(TypeError, "Invalid GPIO direction");
   }
   if (req_data->mode < kGpioModeNone ||
       req_data->mode > kGpioModeOpendrain) {
+    delete req_wrap;
```
Comment (ILyoan) [inline src/iotjs_module_gpio.cpp]: Why don't we have it check range first, then allocate `req_wrap` ?

**Comment by chunseoklee** [comment]
@ILyoan  checking code moved. no need to delete req_wrap.

**Comment by chunseoklee** [comment]
Updated. And modified variable types (```int``` to ```enum```)

**Comment by ILyoan** [comment]
@chunseoklee LGTM

**Comment by chunseoklee** [comment]
rebased and merged

**Pull request closed without merging**